static/js/https/domains.js

```diff
@@ -214,16 +268,16 @@ $(document).ready(function () {
           render: display(names.hsts)
         },
         {
+          data: "https.preloaded",
+          render: display(names.preloaded)
+        },
+        {
           data: "https.grade",
           render: linkGrade
         },
         {
-          data: "Details",
+          data: "",
```

Comment:
No need to show `Details` in front of every message. I'd prefer to get rid of the indentation too, but am not sure how to do that right now.